Extensions/Leaderboards/JsExtension.js

```diff
@@ -86,6 +91,48 @@ module.exports = {
       .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
       .setFunctionName('gdjs.evtTools.leaderboards.hasLastEntrySaveFailed');
 
+    extension
+      .addCondition(
+        'isLeaderboardViewErrored',
+        _('Leaderboard display is errored'),
+        _('Check if the display of the leaderboard errored.'),
+        _('Leaderboard display is errored'),
+        _(''),
+        'JsPlatform/Extensions/leaderboard.svg',
+        'JsPlatform/Extensions/leaderboard.svg'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
+      .setFunctionName('gdjs.evtTools.leaderboards.isLeaderboardViewErrored');
+
+    extension
+      .addCondition(
+        'isLeaderboardViewLoaded',
+        _('Leaderboard display is loaded'),
+        _('Check if the display of the leaderboard has finished loading.'),
+        _('Leaderboard display is loaded'),
```

Comment:
I would even call this "Leaderboard display has loaded and is displayed on screen"